k8s/openebs-operator.yaml

```diff
@@ -131,6 +131,11 @@ spec:
         # to maya install
         - name: OPENEBS_IO_INSTALL_CONFIG_NAME
           value: "maya-install-config-default-0.7.0"
+        # OPENEBS_IO_INSTALL_DEFAULT_CSTOR_POOL decides wheteher default cstor pool should be
+        # configured as a part op openebs installation.
```

Comment:
_n :  op -> of. 
-n : "cstor pool" -> cstor sparse pool"

Reply:
Yes